src/PluginRegistrationDialog.cpp

```diff
@@ -884,6 +897,20 @@ void PluginRegistrationDialog::OnRescan(wxCommandEvent& WXUNUSED(evt))
          reg.Run();
       }
 
+      // Disable all plugins which were previously disabled
+      for (auto& plug : pm.AllPlugins())
+      {
+         PluginType plugType = plug.GetPluginType();
+         if (plugType != PluginTypeEffect && plugType != PluginTypeStub)
+            continue;
+
+         const auto& path = plug.GetPath();
+         if (make_iterator_range(disabledPlugins).contains(path))
```

Comment:
Correct but maybe ineffecient -- I would have looked up in a hash map instead of doing a linear scan of a vector.